<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculateKB</class>
 <widget class="QDialog" name="CalculateKB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btnCalculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInput">
      <attribute name="title">
       <string>Input</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditInput"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProcessing">
      <attribute name="title">
       <string>Processing</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1" columnstretch="1,1">
       <item row="1" column="1">
        <widget class="QTextEdit" name="textEditFitOutput"/>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gLayPlot"/>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
